src/renderer/screens/exchange/Swap2/Form/index.js

```diff
@@ -1 +1,50 @@
-export { default as SwapFormSummary } from "./FormSummary";
+// @flow
+import React, { useState } from "react";
+import SwapFormSummary from "./FormSummary";
+import SwapFormSelectors from "./FormSelectors";
+import Box from "~/renderer/components/Box";
+import styled from "styled-components";
+import ButtonBase from "~/renderer/components/Button";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+import { useTranslation } from "react-i18next";
+
+// SWAP MOCK - PLEASE REMOVE ME ASA LOGIC IS IMPLEMENTED
+const summaryMockedData = {
+  fees: "0.000034 ETH",
+  rate: "1 ETH = 0,06265846 BTC",
+  provider: "Changelly",
+  onProviderChange: () => {},
+  onFeesChange: () => {},
+  onTargetChange: () => {},
+};
+
+const Wrapper: ThemedComponent<{}> = styled(Box).attrs({
+  p: 20,
+})`
+  row-gap: 1.75rem;
+`;
+
+const Button = styled(ButtonBase)`
```

Comment:
You shouldn't need to reimplement over `ButtonBase` at all, what is the reasoning?

Reply:
Design asked for a button with a centred label. As the button from our design system doesn't allow a variant with a centred label (as far as I know, didn't find any props or anything else related to this logic) I have to pass it a CSS rule directly.

Our Button component accepts an inline style object. So I could have pass a style props to the button directly.
```javascript
 <Button primary disabled={!isSwapReady} onClick={onSubmit} style={{ justifyContent: 'center'}}>
 ```
 
However, that looks clearer for me to avoid inline style and rather promote styled components. That's why I created this little one.
```javascript
const Button = styled(ButtonBase)`
  justify-content: center;
`;
 ```
 
The two examples mentioned above do the same thinks but I find the second one more elegant. Do you prefer the first one? What are your feelings about that?

Reply:
I would personally expose a new prop, allowing us to have centered buttons elsewhere, `<Button centered .../>` 🤷🏼‍♂️ but I don't mind, we are going to nuke everything with the rebranding :trollface: so fine by me